public.openapi.yaml

```diff
@@ -979,7 +979,6 @@ components:
             type: string
             maxLength: 63
             minLength: 1
-            pattern: ^[a-z][a-z0-9\-]*$
```

Comment:
Can you please add a comment here to summarise the info in the commit message - so that no one in the future sees this and thinks *"this field should have a validation pattern"*!

Reply:
Updated